gitpod-io/gitpod
# [proxy] Remove nginx server header and update SSL ciphers

How to test:

- ensure the werft build is green
- ensure the in-workspace build works
- open browser `https://aledbf-helm.staging.gitpod-dev.com/workspaces/`

With this change, the http://ssllabs.com/ssltest/analyze.html test:
- Improves the rating from B to A
- Removes weak ciphers
- OCSP is enabled
- Forward Secrecy is enabled

![download](https://user-images.githubusercontent.com/161571/108516088-92eada80-72a4-11eb-8485-4f74432e5897.png)
